packages/zk/build.sh

```diff
@@ -0,0 +1,46 @@
+# http(s) link to package home page.
+TERMUX_PKG_HOMEPAGE=https://github.com/zk-org/zk
+
+# One-line, short package description.
+TERMUX_PKG_DESCRIPTION=" A plain text note-taking assistant"
+
+# License.
+# Use SPDX identifier: https://spdx.org/licenses/
+TERMUX_PKG_LICENSE="GPL-3.0"
+
+# Who cares about package.
```

Comment:
Please remove the comments and empty line between the control variables.
You can take the sample build script:
https://github.com/termux/termux-packages/blob/efa20d206096ce1c299ef2c0e1e3c0f41817e482/sample/build.sh#L5-L11
as a reference.